back/boxtribute_server/models/crud.py

```diff
@@ -9,33 +9,49 @@
 from ..exceptions import BoxCreationFailed, RequestedResourceNotFound
 from .definitions.beneficiary import Beneficiary
 from .definitions.box import Box
+from .definitions.location import Location
 from .definitions.qr_code import QrCode
 from .definitions.x_beneficiary_language import XBeneficiaryLanguage
 from .utils import utcnow
 
 BOX_LABEL_IDENTIFIER_GENERATION_ATTEMPTS = 10
 
 
-def create_box(data):
+def create_box(
+    product_id,
+    location_id,
+    created_by_id,
+    size_id,
+    comment="",
+    items=0,
+    qr_code_code=None,
+):
     """Insert information for a new Box in the database. Use current datetime
     and box state "InStock" by default. Generate an 8-digit sequence to identify the
     box. If the sequence is not unique, repeat the generation several times. If
     generation still fails, raise a BoxCreationFailed exception.
     """
+
```

Comment:
Do you mind adding a sentence to the docstring about the role of location box-state?